include/pybind11/detail/internals.h

```diff
@@ -10,6 +10,7 @@
 #pragma once
 
 #include "../pytypes.h"
+#include "smart_holder_sfinae_hooks_only.h"
```

Comment:
Initially mainly to have an easy way to run the CI with smart_holder as default. But then I thought it's good to keep as an option, to give people an easy way to experiment using smart_holder as default without having to go into their cmake/CI/IDE/whatever they may have. As-is, it works both ways. I'm reluctant to take one option away, especially because the cost is just a few extra lines in smart_holder_sfinae_hooks_only.h. With pytypes.h included first here, everything else is included already anyway.

Reply:
I wasn't asking to remove that option. I was just wondering. For CI, I suggest to use the cmake argument option to allow both checks to run on the same code base. I'm still struggling with some issues in #2930 though... But, I'm optimistic :smile:

Reply:
Oh, good, thanks! And yes, it'll be awesome to trigger all testing with just one PR.